docs/schema.md

```diff
@@ -286,6 +286,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>BITNAMI</code></td>
```

Comment:
Do these advisories come with their own IDs? If so, we can document them under https://ossf.github.io/osv-schema/#id-modified-fields.

Reply:
We had to open a few CVEs in the past, but we don't have a specific identifier like RHSA or similar, so I will remove this from the PR. Thank you for checking!

Reply:
Apologies for my slightly confusing original comment. Nothing having a specific identifier does make things a bit more complicated within the OSV ecosystem, since if you use e.g. CVEs as the ID directly it's hard to disambiguate advisories. 

Do you have any existing examples of any Bitnami-specific advisories? It may be very worthwhile to have a specific identifier for the Bitnami-specific advisories so that it's clear exactly where it comes from and what it's specific to.

Reply:
We only had to register CVEs for specific Bitnami products a few times in the past. And for those we filled a regular CVE on NVD. 

As we are distributing open source programs like WordPress or Redmine as provided from upstream projects (no modification to source code, just easy way to consume), we understand a custom `BITNAMI-2023-0001` ID doesn't make much sense for our specific use case. Can you help us to clarify this point?

Reply:
To clarify, I'm suggesting that Bitnami does have its own ID namespace, to have more ownership over the Bitnami-specific bits. Even if you don't modify source code etc, there may still be interesting additional context through textual descriptions, or different severities. A cheap way to get a custom ID namespace is to just prefix CVEs with BITNAMI or something similar. e.g. BITNAMI-CVE-XXXX-XXX. 

If this doesn't make sense for you, that's OK and we can just go with CVEs. I'll merge this as-is in that case.

Reply:
@oliverchang Thanks for your message, I really appreciate the clarification. Let me check it with the team and get back to you.

Reply:
Yes, absolutely! Please add that ID section back to document this :)

Reply:
Hi @oliverchang,

I have just recovered the section for `BIT` prefix instead of `BITNAMI` as we have agreed. Let me know if there is anything else I may update.